prometheus_hardware_exporter/config.py

```diff
@@ -38,6 +38,7 @@ class Config(BaseModel):
     redfish_discover_cache_ttl: int = DEFAULT_REDFISH_DISCOVER_CACHE_TTL
 
     @validator("port")
+    @classmethod
```

Comment:
Not sure if this changes the behaviour? it looks like these were intended to be classmethods.